eclass/latex-package.eclass

```diff
@@ -105,93 +97,93 @@ latex-package_src_doinstall() {
 
 	# This actually follows the directions for a "single-user" system
 	# at http://www.ctan.org/installationadvice/ modified for gentoo.
-	[ -z "$1" ] && latex-package_src_install all
+	[[ -z ${1} ]] && latex-package_src_install all
 
-	while [ "$1" ]; do
-		case $1 in
+	while [[ ${1} ]]; do
+		case ${1} in
 			"sh")
-				for i in `find . -maxdepth 1 -type f -name "*.${1}"`
-				do
-					dobin $i || die "dobin $i failed"
-				done
+				while read i; do
+					doins ${i}
+				done < <(find . -maxdepth 1 -type f -name "*.${1}")
```

Comment:
So, any reason why

    find -maxdepth 1 -type f -name ... -exec dobin {} +

wouldn't work? Also note it was `dobin` before.

Reply:
just to keep things the same consistence